ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -107,30 +83,9 @@ export const styles = reactStyles({
     fontSize: 28, lineHeight: '34px', color: colors.white, paddingRight: '2.3rem',
     marginTop: '2rem', width: '33%'
   },
-  footerToDelete: {
-    height: '300px', width: '100%', backgroundColor: colors.primary,
-    boxShadow: '0 0 2px 0 rgba(0, 0, 0, 0.12), 0 3px 2px 0 rgba(0, 0, 0, 0.12)',
-    marginTop: '2%',
-  },
-  footerInnerToDelete: {
-    display: 'flex', flexDirection: 'column', marginLeft: '5%', marginRight: '5%',
-  },
-  footerTitleToDelete: {
-    height: '34px', opacity: 0.87, color: colors.white, fontSize: 28,
-    fontWeight: 600, lineHeight: '34px', width: '87.34%', marginTop: '1.4rem'
-  },
-  footerTextToDelete: {
-    height: '176px', opacity: 0.87, color: colors.secondary, fontSize: '16px',
-    fontWeight: 400, lineHeight: '30px', display: 'flex', width: '100%',
-    flexDirection: 'column', flexWrap: 'nowrap', overflowY: 'auto'
-  },
-  linksBlockToDelete: {
-    display: 'flex', marginBottom: '1.2rem', marginLeft: '1.4rem',
-    flexDirection: 'column', flexShrink: 1, minWidth: 0
-  },
   bottomBannerToDelete: {
```

Comment:
Are these other "ToDelete"s ready to delete as well?

Reply:
Not yet! Those are for when we delete the old code for the `enableHomepageRestyle` feature flag.